extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -40,6 +47,14 @@ public function renderInfobox( $text, $params, $parser, $frame ) {
 		return [ $renderedValue, 'markerType' => 'nowiki' ];
 	}
 
+	private function renderUnimplementedTagErrorMesssage( $tagName ) {
+		$renderedValue = '<strong class="error"> '
+			. wfMessage( 'unimplemented-infobox-tag' ) . ': '
```

Comment:
I'm also wondering whether this should be rendered server-side or client-side, as running this will make the message display in the editors language regardless of viewers language (parser cache)